repository/r.json

```diff
@@ -725,6 +725,17 @@
 			]
 		},
 		{
+			"name": "Rekit",
+			"details": "https://github.com/supnate/rekit-sublime",
+			"description": "Provide sidebar menu for Rekit command line tools.",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master"
```

Comment:
This will need to be adjusted per https://packagecontrol.io/docs/submitting_a_package#Step_4